src/AppBundle/Entity/Report/ProfDeputyManagementCost.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace AppBundle\Entity\Report;
+
+use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ * General Management Cost.
+ *
+ * @ORM\Table(name="prof_deputy_management_cost")
+ * @ORM\Entity
+ */
+class ProfDeputyManagementCost
+{
+    /**
+     * ProfDeputyManagementCost constructor.
+     *
+     * @param $profDeputyManagementCostTypeId
+     * @param float $amount
+     * @param string  $hasMoreDetails
+     * @param string  $moreDetails
+     */
+    public function __construct($profDeputyManagementCostTypeId, $amount, $hasMoreDetails, $moreDetails)
+    {
+        $this->profDeputyManagementCostTypeId = $profDeputyManagementCostTypeId;
+        $this->amount = $amount;
+        $this->hasMoreDetails = $hasMoreDetails;
+        $this->moreDetails = $moreDetails;
+    }
+
+    /**
+     * @var int
+     *
+     * @JMS\Groups({"prof-deputy-management-costs"})
+     *
+     * @ORM\Column(name="id", type="integer", nullable=false)
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     * @ORM\SequenceGenerator(sequenceName="prof_management_costs_id_seq", allocationSize=1, initialValue=1)
+     */
+    private $id;
+
+    /**
+     * @var float
+     *
+     * @JMS\Type("float")
+     * @JMS\Groups({"prof-deputy-management-costs"})
+     *
+     * @ORM\Column(name="amount", type="float", precision=14, scale=2, nullable=true)
+     */
+    private $amount;
```

Comment:
this is the only property that needs adding to the trait (but renamed to give it meaning e.g. generalManagementCostAmount or something) re-use the same groups as the rest of the trait uses and it'll bring it back and persist

Reply:
Rest of the properties are not needed, ....sorry :)

Reply:
Gotcha - it did feel a bit strange duplicating all of that 🤦‍♂️